services/core-api/app/api/utils/common_validation_schemas.py

```diff
@@ -0,0 +1,482 @@
+document_schema = {
```

Comment:
Are these actually common or do they have their own models that they're associated with?

Reply:
Maybe the word "common" isn't the right way to describe it. A lot of the schemas in this file get used in multiple different situations though so I put them in this file to avoid duplication. Would you suggest changing the name of the file maybe to helper_validation_schemas?